react/shared/types.ts

```diff
@@ -0,0 +1,50 @@
+export type AffiliationType = {
+  id: string
+  nameAffiliation: string
+  storeId: string
+  salesChannel: string
+  pickupPoints: string
+  glovoId: string
+}
+
+export type SettingDataType = {
```

Comment:
This data structure for the app settings has to match the one [used on the backend](https://github.com/vtex-apps/glovo-integration/blob/a8b4f390b48b6f6289137c933fff853aea255383/node/typings.d.ts#L423).